src/components/portal/portal.tsx

```diff
@@ -86,6 +86,13 @@ export class Portal {
     @Prop({ reflect: true })
     public visible = false;
 
+    /**
+     * The element that the content should be positioned relative to.
+     * Defaults to the limel-portal element.
+     */
+    @Prop()
+    public anchor?: HTMLElement = null;
```

Comment:
I get a build error on this branch 
<img width="543" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/6d8ce300-5b78-4fd2-bf07-0f1a8f8cd187">
Not sure how to solve it

Reply:
Ah! Nice catch!

@jgroth Do you understand why this kind of error seemingly only shows up when running `npm start`? I get that it doesn't show up in the production build, because we're presumably excluding the examples then, but why isn't the docs build failing on this same error? I've had this kind of error that only shows up in `npm start` a few times before.

Reply:
Ah, this is a problem we missed in an earlier PR, so it's already on `next`. I'll fix it there 👍

Reply:
I've fixed it on `next`, and rebased this branch, so it should work now 👍 @Kiarokh

Reply:
⚡